Ct sprint2

FelixChihTing opened the pull request:
the PR addresses [Issue#170](https://github.com/neurodata/brainlit/issues/170)

FelixChihTing committed:
simplify path-searching process

FelixChihTing committed:
Merge branch 'develop' of https://github.com/NeuroDataDesign/brainlit into CTSprint2

FelixChihTing committed:
DataFrame as input to GeometriGraph class

tathey1 reviewed (changes requested):
As far as I can tell, you did not address 1. in [issue 170](https://github.com/neurodata/brainlit/issues/170)

Also, I believe the 4th code block is now useless. If so, delete it.

FelixChihTing committed:
remove session 2

FelixChihTing committed:
Merge branch 'develop' of https://github.com/NeuroDataDesign/brainlit into CTSprint2

FelixChihTing committed:
give path to swc instead of searching by function

FelixChihTing commented:
> 
> 
> As far as I can tell, you did not address 1. in [issue 170](https://github.com/neurodata/brainlit/issues/170)
> 
> Also, I believe the 4th code block is now useless. If so, delete it.

Thank you for your feedback, and I'm sorry. This time I removed the file searching function and directly put the [path to swc](https://github.com/NeuroDataDesign/brainlit/blob/CTSprint2/docs/notebooks/algorithms/view_swc_spline.ipynb?short_path=e43d387#L62-L66).
In the previous change, I actually misunderstood that recursive search=while loop, so I removed the while loop and kept the file searching function. 

The 4th code block contains a few useful lines to define the ```soma```, now the the code block is removed but the [useful lines](https://github.com/NeuroDataDesign/brainlit/blob/CTSprint2/docs/notebooks/algorithms/view_swc_spline.ipynb?short_path=6b08a04#L70-L74) are shifted to the next code block.

FelixChihTing committed:
define variables in the same code block

tathey1 reviewed (changes requested):
- can you use [pathlib](https://realpython.com/python-pathlib/), in the 2nd cell block to identify the swc path? Give it a shot. Currently, your method does not automatically work on Mac.
- The for loop in the second code block is unnecessary, all you need to do is set soma to the first row of the dataframe
- there should be no commented out code, so please remove any such lines e.g. "print(brainlit_path+Brainlit_swc)"
- open PR into [here](https://github.com/neurodata/brainlit), and close this one

FelixChihTing committed:
use pathlib to locate direct path to swc

FelixChihTing committed:
remove for loop when defining soma

FelixChihTing committed:
remove comments
